includes/uiwidgets-ref-target.md

```diff
@@ -1,4 +1,4 @@
-This property accepts one of the following values.
+This property accepts one of the following values:
 
 - A <a href="https://developer.mozilla.org/en-US/docs/Learn/CSS/Introduction_to_CSS/Selectors" target="_blank">CSS selector</a>, or a <a href="https://api.jquery.com/category/selectors/" target="_blank">jQuery selector</a> if you use jQuery
```

Comment:
```suggestion
- A <a href="https://developer.mozilla.org/en-US/docs/Learn/CSS/Introduction_to_CSS/Selectors" target="_blank">CSS selector</a>, or a <a href="https://api.jquery.com/category/selectors/" target="_blank">jQuery selector</a> if you use jQuery.
```

Reply:
This list doesnt't need any dots at the end of the list items (the last two list items are not whole sentences, so as the guideline says, the whole list doesn't need any dots).